### PR TITLE
Ovirt add tagging

### PR DESCRIPTION
**Pull request by joshainglis**
Added the ability to tag VMs using the oVirt module
This pull is dependent on pull #581

### CODE REVIEW

**Comment by gregdek** [comment]
@vincentvdk please review, thanks!

**Comment by bcoca** [comment]
- add 'version_added: "2.0"' for new fields
- remove empty aliases: []

File: cloud/misc/ovirt.py [hunk 1]
```diff
@@ -137,20 +129,35 @@
...
     required: false
-    aliases: []
   state:
     description:
      - create, terminate or remove instances
     default: 'present'
     required: false
-    aliases: []
     choices: ['present', 'absent', 'shutdown', 'started', 'restarted']
+  authorised_key_file:
```
Comment (bcoca) [inline cloud/misc/ovirt.py:143]: i believe it is spelled 'authorized'

Comment (joshainglis) [inline cloud/misc/ovirt.py:143]: Ah yes. Australian spelling :)
I'll switch out the 's'

**Comment by joshainglis** [comment]
Closing this as it is subsumed by #589

**Comment by joshainglis** [comment]
Closing this as it is subsumed by #589

**Pull request closed without merging**